extensions/caffeine/runtime/src/main/java/io/quarkus/caffeine/runtime/graal/CacheConstructorsFeature.java

```diff
@@ -79,6 +79,8 @@ public static String[] typesNeedingConstructorsRegistered() {
                 "com.github.benmanes.caffeine.cache.SSMSA",
                 "com.github.benmanes.caffeine.cache.SSMSW",
                 "com.github.benmanes.caffeine.cache.SSW",
+
+                "com.github.benmanes.caffeine.jcache.copy.JavaSerializationCopier"
```

Comment:
The comment above mentions the cost of registering classes from Caffeine. Is that okay to add this one considering it may not be needed by most projects?

Reply:
I think this one is necessary, without it I get ClassNotFoundException